### PR TITLE
Feature #203293 feat: Feature added to test the course progress repor…

### PR DESCRIPTION
**Pull request by sanjeev-scm**
…t API on dev server

### CODE REVIEW

File: src/app/client/src/app/modules/shared/services/config/url.config.json [hunk 1]
```diff
@@ -204,12 +204,13 @@
     "BATCH": {
       "CREATE": "course/v1/batch/create",
       "UPDATE": "course/v1/batch/update",
-      "ADD_USERS":"course/v1/admin/enroll", 
+      "ADD_USERS": "course/v1/batch/user/add",
```
Comment (komalm) [inline src/app/client/src/app/modules/shared/services/config/url.config.json:207]: @sanjeev-scm @paritshivani can u guys please confirm this change @snehalsabade

File: src/app/client/src/app/modules/shared/services/config/url.config.json [hunk 2]
```diff
@@ -204,12 +204,13 @@
...
       "GET_DETAILS": "course/v1/batch/read",
       "GET_PARTICIPANT_LIST": "course/v1/batch/participants/list",
       "GET_BATCHS": "course/v1/batch/list",
-      "REMOVE_USERS": "course/v1/admin/unenroll"
-    },
+      "REMOVE_USERS": "course/v1/batch/user/remove",
```
Comment (komalm) [inline src/app/client/src/app/modules/shared/services/config/url.config.json:211]: @sanjeev-scm @paritshivani can u guys please confirm this change @snehalsabade

Comment (sanjeev-scm) [reply src/app/client/src/app/modules/shared/services/config/url.config.json:211]: Not updated by me

Comment (snehalsabade) [reply src/app/client/src/app/modules/shared/services/config/url.config.json:211]: I did not update

File: src/app/client/src/app/modules/shared/services/config/url.config.json (hunk 1 above)
Comment (sanjeev-scm) [reply src/app/client/src/app/modules/shared/services/config/url.config.json]: I have updated the paths. Please review and merge

Comment (sanjeev-scm) [reply src/app/client/src/app/modules/shared/services/config/url.config.json]: I have updated the API paths. Please review and merge

**Pull request merged**